cmd/root_test.go

```diff
@@ -19,22 +19,29 @@ import (
 
 func TestRoot_PersistentFlags(t *testing.T) {
 	tests := []struct {
-		name          string
-		args          []string
-		skipNamespace bool
+		name     string
+		args     []string
+		expected bool
 	}{
 		{
-			name: "provided as root flags",
-			args: []string{"--verbose", "--namespace=namespace", "list"},
```

Comment:
Why remove the `--namespace` flag here and below? It seems like this might be removing behavior we want to test, unless I'm reading this incorrectly (always a possibility!).

Reply:
Good question.  You can see from the handfull of other open PRs I am adding more deliberate usage and tests to the commands which utilize namespace as well.

I am systematically moving away from globally shared flags for a couple of reasons:  first, the help text is much better off being custom tailored to the command for which the flag applies.  Second, it is a bit complicated to test the flag is being used properly, because tests have to first instantiate a root command object.  Finally, since the namespace really isn't applicable to every command, it seems sloppy to have it included in the help text.

I can see two flags beng truly globally shared:  "verbose" and "output".  The others seem like more confusion than it's worth.